SPECS/kernel/kernel.spec

```diff
@@ -249,6 +249,7 @@ ln -s vmlinux-%{uname_r} %{buildroot}%{_libdir}/debug/lib/modules/%{uname_r}/vml
 cat > %{buildroot}/boot/linux-%{uname_r}.cfg << "EOF"
 # GRUB Environment Block
 mariner_cmdline=init=/lib/systemd/systemd ro no-vmw-sta crashkernel=256M
+mariner_cmdline=init=/lib/systemd/systemd ro crashkernel=256M
```

Comment:
Did you mean to overwrite the line above?

Reply:
This does look like a duplicate line. Vince can you confim that it's intentional?